packages/headless/src/features/commerce/common/actions.ts

```diff
@@ -46,7 +40,19 @@ export interface QueryCommerceAPIThunkReturn {
   analyticsAction: PreparableAnalyticsAction<StateNeededByQueryCommerceAPI>;
 }
 
-export const buildBaseCommerceAPIRequest = async (
+export const buildCommerceAPIRequest = async (
```

Comment:
I just moved this function to the top since it's the exported one, no code change here otherwise